jraph/_src/utils.py

```diff
@@ -1122,3 +1122,32 @@ def wrapper(graph: gn_graph.GraphsTuple) -> gn_graph.GraphsTuple:
     return zero_out_padding(graph_net(graph))
 
   return wrapper
+
+  
```

Comment:
please leave only one line break

Reply:
Done, but just curious when should you leave one line break and when should you leave two? Because in `utils.py`, I think all have two line breaks?

Reply:
Sorry, this one was my bad when copying over, this really is the last one, I should give you some sort of kudos for being so patient with me :)

Reply:
No worries at all :)

So do you want me to make it two line breaks again?

Reply:
yes please, thank you!

Reply:
Done and pushed!